app/src/main/java/com/udacity/bakingapp/fragment/RecipeStepFragment.java

```diff
@@ -131,6 +134,7 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
 
     private void initializePlayer(String mediaURL) {
         if (mExoPlayer == null) {
+            Timber.d("Setup exoplayer" + mPlayerWindow + " " + mPlayerPosition + " " + mPlayerAutoplay);
```

Comment:
nit: This seems like leftover log statements from debugging. Should be removed if no longer used.